cluster/gce/config-default.sh

```diff
@@ -98,6 +98,7 @@ export KUBELET_TEST_ARGS=${KUBE_KUBELET_EXTRA_ARGS:-}
 
 # Default container runtime
 export CONTAINER_RUNTIME=${KUBE_CONTAINER_RUNTIME:-containerd}
+export CONTAINER_RUNTIME="docker"
```

Comment:
@thockin this commit should indeed not merge, it's just a quick way to ensure the default presubmit runs dockershim for vetting the PR by forcing it regardless of the config.

Reply:
(also I suggested this as a low effort way to sanity check the code 🙃 )
I'll keep an eye out to help merge when it's removed.

Reply:
removed